training/.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
```

Comment:
These files should be in the root folder, not inside a `training` folder. Here you have added a folder called `training` where inside of it is the code of the project, but everything should go in the root level of the repository (the folder that is immediately created when you clone the project).